src/lunabot_bringup/test/test_oakd_camera_contract.py

```
@@ -0,0 +1,88 @@
+    assert remappings == [
+        ("oak_front/rgb/image_raw", "/camera_front/image"),
+        ("oak_front/rgb/camera_info", "/camera_front/camera_info"),
+    ]
+
+
+def test_camera_launch_arguments_keep_depth_optional_and_usb3_first():
+    launch_args = camera_launch_arguments(
+        driver_name="oak_front",
+        camera_model="OAK-D-PRO",
+        parent_frame="camera_front_link",
+        params_file="/tmp/oakd_front.yaml",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/oakd_front.yaml"

(S108)